src/shogun/evaluation/CrossValidationStorage.cpp

```diff
@@ -260,7 +143,7 @@ bool CrossValidationStorage::operator==(const CrossValidationStorage& rhs) const
 {
 	auto member_vars = m_num_runs == rhs.m_num_runs &&
 	                   m_num_folds == rhs.m_num_folds &&
-	                   m_expose_labels->equals(rhs.m_expose_labels);
+	                   m_original_labels->equals(rhs.m_original_labels);
```

Comment:
why do we need this operator?

Reply:
I guess it was needed for some old unit test which is not there anymore. I'll check if I can remove it safely...

Reply:
They can be removed indeed.